<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>271</width>
    <height>215</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>15</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>15</number>
    </property>
    <property name="bottomMargin">
     <number>15</number>
    </property>
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Temperatura:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="tempIn"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Convertir a:</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QRadioButton" name="cenKel">
         <property name="text">
          <string>Centigrados a Kelvin</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="kelCen">
         <property name="text">
          <string>Kelvin a Centigrados</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="cenFah">
         <property name="text">
          <string>Centigrados a Fahrenheit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="fahCen">
         <property name="text">
          <string>Fahrenheit a Centigrados</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelTemp">
      <property name="text">
       <string>Temperatura:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnCalcular">
      <property name="text">
       <string>Calcular</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
